test/helper_modules.rb

```diff
@@ -48,11 +63,13 @@ def self.load_test_categories_and_transaction_types_to_db(community)
 
     def self.load_categories_and_transaction_types_to_db(community, transaction_types, categories)
       # Load transaction types
-      transaction_types.each do |type, translations|
+      transaction_types.each do |type, opts|
 
         transaction_type = Object.const_get(type.to_s).create!(:type => type, :community_id => community.id)
+        transaction_type.create_transaction_process(process: opts[:process])
```

Comment:
It'd be better to use ```create_transaction_process!``` as it throws instead of returning false on invalid association object.